### ISSUE TITLE
Proposition déplacement du marqueur carto

### ISSUE THREAD

**Issue by TheoLechemia** [PnX-SI/gn_mobile_occtax#176]
**Type d'amélioration**
ergonomie 

**Proposition**
Plusieurs utilisateurs m'ont fait remonter leur difficulté à déplacer le marqueur sur la carte avec précision. Avec des gros doigts, ce n'est visiblement pas si évident (le marqueur est caché derrière doit et on ne voit pas exactement ce que l'on pointe).
Plusieurs applications ont une ergonomie différente que je trouve intéressante (dont google maps et naturalist) : le pointer est fixe au centre de la carte (on ne peut pas le bouger), mais il se déplace en même temps qu'on se déplace sur celle-ci : 

https://user-images.githubusercontent.com/15265745/175905732-1b52e76e-0386-4f3b-a812-aafefb278a19.mp4

**Comment by JeromeMaruejouls** [comment]
Oui, ce fonctionnement me paraît bien. Autre solution, comme par exemple dans OruxMaps : décaler le pointeur un petit peu vers le haut lorsque l'on passe en édition par appui long, pour éviter que le doigt cache l'endroit précis où l'on veut positionner le marqueur.

À voir quelle solution est la plus intuitive et la plus rapide à implémenter.

**Comment by gildeluermoz** [comment]
Il avait aussi été proposé de matérialiser le centre de la carte par une
petite cible discrète et présente en permanence. Celle-ci permettrait de
voir précisément où va se placer le marqueur.

Le lun. 27 juin 2022 à 12:43, DonovanMaillard ***@***.***> a
écrit :

**Comment by camillemonchicourt** [comment]
Depuis la version 2.3.0, on peut maintenant ajouter le marqueur ou le déplacer directement en faisant un clic long où on veut sur la carte : https://github.com/PnX-SI/gn_mobile_occtax/issues/14.

**Comment by camillemonchicourt** [comment]
Mis en place dans la 2.8.
On a gardé le système existant où on peut placer ou déplacer le marqueur avec le doigt, mais en ajoutant la possibilité de le localiser au centre de la carte, représenté par une petite croix au centre de l'écran : 

https://github.com/user-attachments/assets/33f7406e-e08a-44a0-8419-0ad330c3b86c
